lib/actual_db_schema/commands/rollback.rb

```diff
@@ -4,10 +4,15 @@ module ActualDbSchema
   module Commands
     # Rolls back all phantom migrations
     class Rollback < Base
+      def initialize(manual_mode)
```

Comment:
```suggestion
      def initialize(manual_mode: false)
```

And change the callers accordingly please.